client/src/components/UserPage.tsx

```diff
@@ -150,18 +151,47 @@ const UserPage = ({ match }: RouteComponentProps<RouteInfo>) => {
         </Callout>
       )}
       <UserCard user={user} userId={userId} removeButton={false} linkButton />
-      {numOfItems > 0 && (
+      {numOfItems > 0 && transactions.length === 0 && (
+        <div className="loading">
+          <LoadingSpinner />
+          <Callout>
```

Comment:
Personally I didn't see the need to include this message here while we're loading net worth. It kind of freaked me out because I saw a message in a box with a link and then it disappeared and I wondered if I had missed something important, so I went back to relogin just to make sure I could read the message. For someone that doesn't notice it, or is slow to read it before it disappears, it could be a bit confusing. Not asking that we make a change, just leaving my feedback.